src/components/Puppies.jsx

```diff
@@ -2,35 +2,35 @@ import React from "react";
 import { useContext } from "react";
 import { TransactionContext } from "../context/TransactionContext";
 
-const PuppyCard = ({ id, name, imageUrl }) => {
+const PuppyCard = ({ dogId, imageUrl, name, birthday, description}) => {
     const handleDonate = () => { }
 
     return (
-        <div className="bg-[#181918] m-4 flex flex-1 2xl:min-w-[450px] 2xl:max-w-[500px] sm:min-w-[270px] sm:max-w-[300px]
+        <div className="bg-[#181918] m-4 flex flex-1 2xl:min-w-[300px] 2xl:max-w-[350px] sm:min-w-[270px] sm:max-w-[300px]
                          min-w-full flex-col p-3 rounded-md hover:shadow-2xl"
         >
             <div className="flex flex-col items-center w-full mt-3">
                 <div className="display-flex justify-start w-full mb-6 p-2">
                     <p className="text-white text-base">Name: {name}</p>
+                    <p className="text-white text-base">Birthday: {birthday}</p>
+                    <p className="text-white text-base">Description: {dogId}</p>
```

Comment:
```suggestion
                    <p className="text-white text-base">Description: {description}</p>
```

[Question]

Should we use `description` here instead of `dogId`?